ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -102,6 +106,95 @@ def _get_table_description(schema: str, table: str, inspector: Inspector) -> str
     return description
 
 
+def get_dbt_local(config):
+    if config.dbtConfig.dbtCatalogFilePath is not None:
+        with open(
+            config.dbtConfig.dbtCatalogFilePath, "r", encoding="utf-8"
+        ) as catalog:
+            dbt_catalog = catalog.read()
+    if config.dbtConfig.dbtManifestFilePath is not None:
+        with open(
+            config.dbtConfig.dbtManifestFilePath, "r", encoding="utf-8"
+        ) as manifest:
+            dbt_manifest = manifest.read()
+    return json.loads(dbt_catalog), json.loads(dbt_manifest)
```

Comment:
I think this would fail if the paths are not properly informed 🤔  Here we should return a Tuple of Optional, returning the json.loads or None. Then, the logic afterward should validate that everything is informed

Reply:
okay got it